src/main/java/com/tira/restaurants/authlisteners/AuthenticationSuccessEventListener.java

```diff
@@ -0,0 +1,31 @@
+package com.tira.restaurants.authlisteners;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationListener;
+import org.springframework.security.authentication.event.AuthenticationSuccessEvent;
+import org.springframework.security.web.authentication.WebAuthenticationDetails;
+import org.springframework.stereotype.Component;
+
+import com.tira.restaurants.service.LoginAttemptService;
+
+@Component
+public class AuthenticationSuccessEventListener implements ApplicationListener<AuthenticationSuccessEvent> {
```

Comment:
I will probably delete this part, I created this EventListener in order to reset user login attempts to zero. But since I will not be blocking users by IP I will probably delete this part.